docs/install/create-project-cookieplone.md

```diff
@@ -0,0 +1,312 @@
+---
+myst:
+  html_meta:
+    "description": "Create a Plone project with the Volto frontend (development or pre-release)"
+    "property=og:description": "Create a Plone project with the Volto frontend (development or pre-release)"
+    "property=og:title": "Create a Plone project with the Volto frontend (development or pre-release)"
+    "keywords": "Plone, Plone 6, Volto, create, project, install, cookieplone"
+---
+
+
+(create-project-cookieplone-label)=
+
+# Create a project with Volto (development or pre-release)
+
+This chapter describes how you can create a web application using the latest **development release** version of Plone with **Volto 18 or later** for the frontend, while having full control over its development and deployment.
+
+```{seealso}
+For other installation options, see {doc}`/install/index`.
+```
+
+```{versionadded} Volto 18.0.0-alpha.43
+Cookieplone is now the method to create a Plone project with unstable versions of Volto, version 18.0.0-alpha.43 and above.
+```
+
+
+## System requirements
+
+Plone 6 has both hardware requirements and software pre-requisites.
+
+
+### Hardware requirements
+
+```{include} /_inc/_hardware-requirements.md
+```
+
+### Pre-requisites for installation
+
+```{include} ../volto/contributing/install-operating-system.md
+```
+
+-   Python {SUPPORTED_PYTHON_VERSIONS}
+-   {term}`pipx`
+-   {term}`nvm`
+-   {term}`Node.js` LTS 20.x
+-   {term}`GNU make`
+-   {term}`Git`
+
+
+#### Python
+
+Installing Python is beyond the scope of this documentation.
+However, it is recommended to use a Python version manager, {term}`pyenv` that allows you to install multiple versions of Python on your development environment without destroying your system's Python.
+Plone requires Python version {SUPPORTED_PYTHON_VERSIONS}.
+
+
+#### pipx
+
+Install {term}`pipx`.
+
+```shell
+pip install pipx
+```
+
+
+#### nvm
+
+```{include} ../volto/contributing/install-nvm.md
+```
+
+
+#### Node.js
+
+```{include} ../volto/contributing/install-nodejs.md
+```
+
+[do we need to enable corepack?]
```

Comment:
````suggestion
Enable {term}`corepack` so that Node.js will take care of installing {term}`pnpm` as a package manager.

```console
corepack enable
```
````